frontend/test/__support__/server-mocks/table.ts

```diff
@@ -1,11 +1,14 @@
 import fetchMock from "fetch-mock";
-import type { Table } from "metabase-types/api";
+import type { ForeignKey, Table } from "metabase-types/api";
 import { setupFieldEndpoints } from "./field";
 
-export function setupTableEndpoints(table: Table) {
+export function setupTableEndpoints(
+  table: Table,
+  foreignKeys: ForeignKey[] = [],
+) {
   fetchMock.get(`path:/api/table/${table.id}`, table);
   fetchMock.get(`path:/api/table/${table.id}/query_metadata`, table);
-  fetchMock.get(`path:/api/table/${table.id}/fks`, []);
+  fetchMock.get(`path:/api/table/${table.id}/fks`, foreignKeys);
```

Comment:
This is used in `frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx`